### PR TITLE
Fix build pipeline

### PR DESCRIPTION
**Pull request by adrianvmsft**
Updating the build pipeline to meet the latest security requirements:
- Removing dependency on myget feed
- Consuming signed packages by approved owners
- Moving to scaleset pool
- Fixing the build getting stuck while trying to deploy the vsix to Visual Studio

### CODE REVIEW

**Review by chzuluag — APPROVED** [review]

File: src/Microsoft.VisualStudio.SlowCheetah.Tests/Microsoft.VisualStudio.SlowCheetah.Tests.csproj [hunk 1]
```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="MicroBuild.NonShipping" Version="2.0.54" />
+    <PackageReference Include="Microsoft.VisualStudio.Internal.MicroBuild" Version="2.0.66" />
```
Comment (AArnott) [inline src/Microsoft.VisualStudio.SlowCheetah.Tests/Microsoft.VisualStudio.SlowCheetah.Tests.csproj:15]: This is not an equivalent change. It *was* the `NonShipping` package and now it's not. You should still be able to find a non-shipping variant of this package under the new naming scheme.
Same for your other test projects.

Comment (adrianvmsft) [reply src/Microsoft.VisualStudio.SlowCheetah.Tests/Microsoft.VisualStudio.SlowCheetah.Tests.csproj:15]: Thanks for the heads up! I fixed it in #236

Comment (AArnott) [reply src/Microsoft.VisualStudio.SlowCheetah.Tests/Microsoft.VisualStudio.SlowCheetah.Tests.csproj:15]: I only see one test project fixed in that PR. But in _this_ PR the bug was introduced in at least two test projects.

Comment (adrianvmsft) [reply src/Microsoft.VisualStudio.SlowCheetah.Tests/Microsoft.VisualStudio.SlowCheetah.Tests.csproj:15]: Yep, I didn't notice there were 2 of them. Fixed the other one in #237

**Pull request merged**